tensorflow/lite/micro/tools/python_interpreter/src/interpreter_wrapper_pybind.cc

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <pybind11/pybind11.h>
+
+#include "tensorflow/lite/micro/tools/python_interpreter/src/interpreter_wrapper.h"
+
+namespace py = pybind11;
+using tflite::InterpreterWrapper;
+
+PYBIND11_MODULE(interpreter_wrapper_pybind, m) {
+  m.doc() = "pybind11 TFLM interpreter";
```

Comment:
nit: should it be just TFLM interpreter?

Reply:
I'm not actually sure where is string is surfaced. I believe TFLite doesn't even define it. I can make it "TFLM interpreter" to be more concise.